### ISSUE TITLE
arxivist interfering with arxiv user login?

### ISSUE THREAD

**Issue by mattia-vaccari** [dfm/arxivist#1]
Since I've installed arxivist (on Chrome, on Mac High Sierra) I can't seem to be able to log into my arxic account because of the following error.
---
This page isn’t working arxiv.org redirected you too many times.
Try clearing your cookies.
ERR_TOO_MANY_REDIRECTS
---
Note that user login works fine on Safari at the same time...

**Comment by dfm** [comment]
Thanks for reporting this! For now, you can temporarily disable the extension while you log in by going to chrome://extensions in your browser. I'll try to figure out what might be causing this.

**Comment by mattia-vaccari** [comment]
yep, i also tried that and that fixed it, i just thought you may want to be aware of the issue and perhaps note it in the README... thanks!

**Comment by dfm** [comment]
Definitely. Thanks for reporting this!
